recipes/timers-and-such/direct/hparams/train_with_wav2vec.yaml

```diff
@@ -0,0 +1,199 @@
+# ############################################################################
+# Model: Direct SLU with WaV2VEC
+# Encoder: Pre-trained Wav2vec
+# Decoder: GRU + beamsearch
+# Tokens: BPE with unigram
+# losses: NLL
+# Training: Timers and Such
+# Authors:  Abdelmoumene Boumadane, AbdelWahab Heba, Loren Lugosch 2021
+# ############################################################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+experiment: wave2vec_classification_seq2seq
```

Comment:
wav2vec*, why you call it classification_seq2seq ?
maybe: `wav2vec-train-real`

Reply:
Yes I'll change it